### ISSUE TITLE
Blast error in v2.0.2

### ISSUE THREAD

**Issue by stitam** [katholt/Kaptive#22]
Hi,

Many thanks for developing Kaptive! While v2.0.0 works as intended, I receive the following error message when using v2.0.2:

```
Error: makeblastdb encountered an error:
Error: Unknown argument: "blastdb_version"
Error:  (CArgException::eInvalidArg) Unknown argument: "blastdb_version"
``` 
I am using blast v2.3.0 as recommended. 

I was wondering if there was a change between v2.0.0 and v2.0.2 in the blast dependency as well? Many thanks.

**Comment by kelwyres** [comment]
Hi, 

That's odd - no, there's been no change to the dependency - just the Ab database files and documentation.

Thanks

**Comment by rrwick** [comment]
Kelly is correct that the dependency didn't officially change, but I see that a commit intended to fix support for Debian (42c82a6, #20) added `-blastdb_version 4` to the `makeblastdb` call. This option was added around BLAST v2.9.0 or so, which is why you hit that error.

Seeing as how the default value for `-blastdb_version` is `4`, I have removed that option which should resolve the problem in the latest Kaptive release: https://github.com/katholt/Kaptive/releases/tag/v2.0.3

Thanks!
